lib/src/widgets/yust_text_field.dart

```diff
@@ -174,8 +174,7 @@ class _YustTextFieldState extends State<YustTextField> {
   @override
   Widget build(BuildContext context) {
     final textValue = widget.value ?? '';
-    if (textValue != _initValue &&
-        textValue != _controller.text &&
+    if ((textValue != _initValue || textValue != _controller.text) &&
```

Comment:
Previously, in an updated text field, the _initValue used in the state was not updated correctly. Thus, the _initValue was the same after changing the text from the keyboard. Now, when a data update occurred that returned the text to its original state, the update was not displayed (even if it was present in the database).

Example:
Text in database: 'Test123'
After opening the record:

| textValue | _initValue | _controller.text | | comment |
| -------- | -------- | -------- | -------- | -------- |
|  N/A*     | Test123 | Test123 || next action: type in "Test1234" |
|  Test1  | Test123 | Test1 || Test1 == Test1 -> If evaluates to false -> _initValue is not updated; _controller.text is not altered -> Test1 is shown in the Textfield; next action: reverse Text to Test123** |
|  Test123  | Test123 | Test1 || Test123 == Test123 -> _controller.text is not updated -> It still shows Test1, even if the database contains "Test123". This only updates after a complete refresh |


*the textValue variable only exists in the build function
** e.g. through the transfer function in the record picker